beacon/endpoints/rest/schemas/__init__.py

```diff
@@ -5,13 +5,13 @@
 
 supported_schemas = {
     # default
-    'beacon-info-v2.0.0-draft.2': default.beacon_info_v30,
-    'beacon-dataset-v2.0.0-draft.2': default.beacon_dataset_info_v30,
-    'beacon-variant-v2.0.0-draft.2': default.beacon_variant_v30,
-    'beacon-variant-annotation-v2.0.0-draft.2': default.beacon_variant_annotation_v30,
-    'beacon-biosample-v2.0.0-draft.2': default.beacon_biosample_v30,
-    'beacon-individual-v2.0.0-draft.2': default.beacon_individual_v30,
-    'beacon-cohort-draft3.1': default.beacon_cohort_v31,
+    'beacon-info-v2.0.0-draft.3': default.beacon_info_v30,
+    'beacon-dataset-v2.0.0-draft.3': default.beacon_dataset_info_v30,
+    'beacon-variant-v2.0.0-draft.3': default.beacon_variant_v30,
+    'beacon-variant-annotation-v2.0.0-draft.3': default.beacon_variant_annotation_v30,
+    'beacon-biosample-v2.0.0-draft.3': default.beacon_biosample_v30,
+    'beacon-individual-v2.0.0-draft.3': default.beacon_individual_v30,
+    'beacon-cohort-v2.0.0-draft3': default.beacon_cohort_v31,
```

Comment:
This is missing a `.` between `draft` and `3`. And I'd keep the `3.1` as I think it's the correct one, right? I mean, it's including things that are not in version `draft.3` of this schema (e.g. the `id` and `label` for ontology fields).